components/dashboard/src/settings/Account.tsx

```diff
@@ -7,23 +7,40 @@
 import { User } from "@gitpod/gitpod-protocol";
 import { useContext, useState } from "react";
 import Modal from "../components/Modal";
+import { getGitpodService } from "../service/service";
 import { UserContext } from "../user-context";
 import { SettingsPage } from "./SettingsPage";
 
 export default function Account() {
     const { user } = useContext(UserContext);
 
     const [modal, setModal] = useState(false);
+    const [typedEmail, setTypedEmail] = useState('');
+
+    const primaryEmail = User.getPrimaryEmail(user!);
+
+    const deleteAccount = async () => {
+        await getGitpodService().server.deleteAccount();
+        //TODO we are sad to see you go view
```

Comment:
@gtsiolis I made it so that users get redirected to the home page. 
We should think about something that helps users reconsidering their decision. 
That said probably something to postpone to after this milestone.

Reply:
Current flow looks good but yes, we could improve this. Some visual feedback (see _alert component_) on the login page that the account has been deleted could be a starting point. I'd also vote for leaving this out of the scope of this milestone. Added https://github.com/gitpod-io/gitpod/issues/3554 as a follow-up issue.